service/core/client/src/main/scala/com/lightbend/lagom/internal/client/WebSocketClient.scala

```diff
@@ -41,11 +42,13 @@ import scala.collection.immutable
 /**
  * A WebSocket client
  */
-private[lagom] abstract class WebSocketClient(environment: Environment, eventLoop: EventLoopGroup,
+private[lagom] abstract class WebSocketClient(environment: Environment, config: Config, eventLoop: EventLoopGroup,
```

Comment:
Hi @datalchemist , this PR looks great. I would only suggest a small change. Creating a new namespace in the configuration like `lagom.client.websocket` could be accompanied by a strongly typed `WebSocketClientConfig` class that only provides access to specific settings.

Reply:
Good. I was quite unsure what was the correct way to provide the Config. So I've corrected this point as you suggested.

Thanks for your detailed feedback!

Reply:
Is it possible you have commits locally that you still haven't pushed yet? I still see `config: Config` in the parameter list in `WebSocketClient` and `ScaladslWebSocketClient`.

Reply:
Great suggestion, @ignasi35. We are currently inconsistent about this in Lagom, but I think it would be a great convention to converge on providing typed config classes everywhere. It makes reuse (including in tests) much easier if you don't force coupling to a `Config` object.

Reply:
@ignasi35 Actually, the changes I made last week was only to use the LagomConfigComponent instead of a simple abstract config declaration.
Anyway, now, I've modified the code to use a typed WebSocketClientConfig object.